src/test/functional/features/update-contact-details.feature

```diff
@@ -0,0 +1,74 @@
+Feature: Update contact details
+  Background: Logged in for hub page
+    Given I create a new user and login
+    And I've already completed the form using the fixture "completeCase"
+
+  Scenario: Applicant 1 updates contact details
+    And I go to '/check-your-answers'
+    And I click "I confirm"
+    And I click "I believe that the facts stated in this application are true"
+    When I click "Continue to payment"
+    And I pay and submit the application
+    Then the page should include "Application submitted"
+    Given a case worker issues the application
```

Comment:
can we leave a line before each 'Given', might make it easier to follow

Reply:
Maybe its easier if we use When, i dont like using a gap, doesnt feel like its part of the test but another section.